tests/utils.go

```diff
@@ -4104,7 +4112,7 @@ func HasLiveMigration() bool {
 	return HasFeature("LiveMigration")
 }
 
-func StartTCPServer(vmi *v1.VirtualMachineInstance, port int) {
+func StartTCPServer(vmi *v1.VirtualMachineInstance, port int, isFedoraVM bool) {
```

Comment:
Is `isFedoraVM` still needed here?

Reply:
The test is defined in a [table](https://github.com/kubevirt/kubevirt/pull/3249/files#diff-0836acab6741ba8251f478229fe81a6dR49), and the 'servers' are [table parameters](https://github.com/kubevirt/kubevirt/pull/3249/files#diff-0836acab6741ba8251f478229fe81a6dR85-R86) - e.g. they must share the same function signature. 

I can get rid of the table and have two tests if you prefer. that is probably more readable.

Reply:
Well, if it's specifically needed for a test, normally you could just define a wrapper function with a required signature.
But the real function won't be affected this way.

Reply:
RIght. Added a function wrapping over the ' tests.StartTCPServer` function, sharing a common signature w/ `tests.StartHTTPServer` .

Thanks for pointing it out.